pkg/api/testing/fuzzer.go

```diff
@@ -227,6 +227,12 @@ func FuzzerFor(t *testing.T, version string, src rand.Source) *fuzz.Fuzzer {
 				c.Fuzz(&e.Count)
 			}
 		},
+		func(s *api.Secret, c fuzz.Continue) {
+			c.Fuzz(&s.TypeMeta)
+			c.Fuzz(&s.ObjectMeta)
+
+			s.Type = api.SecretTypeOpaque
```

Comment:
This is a minor thing, but it would really be nice to have a comment here explaining why this is sufficient.  It took me a while to think about it.